### ISSUE TITLE
Unable to find Gyroflow app path. Make sure to run Gyroflow app at least once and that version is at least v1.4.3

### ISSUE THREAD

**Issue by pljeferson** [gyroflow/gyroflow-ofx#55]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you tried the latest build?

- [X] I have tried the latest build

### Do you have latest GPU drivers installed?

- [X] I have the latest GPU drivers installed, I'm absolutely sure.

### Have you checked the documentation?

- [X] I have read the FAQ and Troubleshooting and didn't find my issue there.

### Gyroflow version

v.1.5.4

### What operating system are you using?

Mac OS - Sonoma 14.0

### What GPU are you using?

Apple Silicon M1

### What happened?

NLE: Davinci Resolve Studio 18.6.1 Build 8

After importing my clip into the Timeline and applying the OFX Plugin to the clip. After click for "Load for current file" and then when i try to "Open in Gyroflow" I can't access the program, the Gyroflow won't start even though I have already used it standalone before. 

I just get a message: "Unable to find Gyroflow app path. Make sure to run Gyroflow app at least once and that version is at least v1.4.3"

![IMG_0030](https://github.com/gyroflow/gyroflow/assets/151591630/9a3c1ba4-fe78-48b0-991a-cdb94bea3221)
![IMG_0029](https://github.com/gyroflow/gyroflow/assets/151591630/9cf85af8-643c-4efa-82ad-346b5f8a27ee)

**Comment by pljeferson** [comment]
> Do you have the main Gyroflow application in your `/Applications` folder?

Yes, i have it in my **/Applications** folder. I did the installation directly from the App Store.

**Comment by pljeferson** [comment]
> use latest dev build of the plugin from https://gyroflow.xyz/devbuild/ also if using plugin, install Gyroflow from .dmg instead of app store, because of app sandbox limitations

Perfect, I will try now and then I will comeback with any feedback

**Comment by pljeferson** [comment]
> use latest dev build of the plugin from https://gyroflow.xyz/devbuild/ also if using plugin, install Gyroflow from .dmg instead of app store, because of app sandbox limitations

Okay, in fact Installing Gyroflow from .dmg instead of app store solved the problem when opening clicking "Open in Gyroflow" buttom as the default page Open right away, but now it's become another problem cause the "Load for current file" does not work anymore, even already toogle ON "External scripting using" to "Local" in Preferences -> General.

Tried switch "Local" to "None" to go back to Local and vice versa, restarting the program...but no sucess

I know the External scripting function only works in the paid version of DR Resolve, but already use that paid.
(Davinci Resolve Studio 18.6.1 Build 8)

![IMG_0031](https://github.com/gyroflow/gyroflow-ofx/assets/151591630/322f32f2-44b7-49fd-852a-fc0e3895f5f3)
![IMG_0033](https://github.com/gyroflow/gyroflow-ofx/assets/151591630/ba0d0f14-3381-4986-98ba-b78db8816084)

**Comment by SylvainLopezVisuals** [comment]
I have exactly the same problem having scrupulously followed all the instructions previously given. "Load for current file" still doesn't work, and still gives me the same message.

**Comment by shawnclovie** [comment]
I just met the same issut.
I was installed Gyroflow 1.5.2, then renamed it to Gyroflow-v1.5.2.app in /Application, and copy the version 1.5.4 into /Application.

After I installed the plugin and click Open in Gyroflow, Gyroflow-v1.5.2.app present to me.
Then I deleted Gyroflow-v1.5.2.app, the Open in Gyroflow did not work.
Then I ran these:
> rm -f $HOME/Library/Preferences/com.gyroflow-xyz.Gyroflow.plist ; killall cfprefsd
Then "Unable to find Gyroflow app path. Make sure to run Gyroflow app at least once and that version is at least v1.4.3" would present while clicked the button.

Button "Load for current file" is working fine.
